test/index.test.js

```diff
@@ -325,10 +325,11 @@ test('test catch of invalid image data error', (done) => {
     .use(sharp())
     .build((err, files) => {
       if (err) {
+        console.log(err);
         expect(
           err
             .toString()
-            .indexOf('Input buffer contains unsupported image format')
+            .indexOf('Input Buffer is empty')
```

Comment:
Can we change the buffer content to sth like `this is not image data but some mocked text`?


Than the old error check should be fine again?

Reply:
I'll try that and if that works change it so the test check stays the same.